finagle-core/src/main/scala/com/twitter/finagle/service/FailureAccrualFactory.scala

```diff
@@ -316,18 +335,9 @@ class FailureAccrualFactory[Req, Rep](
           // ProbeClosed state. The result of first to complete will determine
           // whether the factory transitions to Alive (successful) or Dead
           // (unsuccessful).
-          state match {
-            case ProbeOpen =>
-              probesCounter.incr()
-              self.synchronized {
-                state match {
-                  case ProbeOpen => state = ProbeClosed
-                  case _ =>
-                }
-              }
-            case _ =>
-          }
+          stopProbing()
 
+          // Invoke service
```

Comment:
This comment isn't giving the reader any additional information, I think we can drop it.